Makefile

```diff
@@ -93,6 +93,14 @@ integration-test: ## Run the integration tests
 	$(MAKE) start-db
 	${DOCKER_COMPOSE} run --rm app npm run test:integration; exit=$$?; ${DOCKER_COMPOSE} down; exit $$exit
 
+api-validate: ## Run the API analysis
+	$(MAKE) start wait-healthy
+	docker run --rm --network host hydrofoil/hydra-analyser:0.2.0 http://localhost:8080/; exit=$$?; ${DOCKER_COMPOSE} down; exit $$exit
+
+api-test: ## Run the API tests
```

Comment:
API tests says that these tests exercise the system through the API, but nothing specific about how they take advantage of hypermedia and/or what parts are generated/discovered. Is this the fairest description?

Reply:
Could call it system testing?

Reply:
Your call, would add something a qualifier like `hypermedia-api-testing` if that's the main aspect.

Reply:
Will keep as is. The main part of the app is the API, and the API is hypermedia, so don't think it's necessary to include it here.

Reply:
:+1: